### DISCUSSION TITLE
v0.1

### DISCUSSION THREAD

**Discussion by c3d** [c3d/db48x#19] [Announcements]
First preview of the firmware with pre-built binaries

To install:

1. Put your DM42 in USB disk mode
2. Expand the contents of the release at the base of the disk. You should have four files, `DB48X.pgm`, `DB48X_qspi.bin` `help/DB48X.md` and `STATE/test.48S`
3. Using the calculator's setup menu, flash QSPI image from USB disk. This will load `DB48X_qspi.bin` and erase it from the disk.
4. Using the calculator's setup menu, load the `DB48X.pgm`.

Note: Normally, the QSPI binary is "compatible" with the original DM42 program, so that you can switch back and forth. There may be limitations if the symbols in that file are not at the right address. If your DM42 program misbehaves after installing this QSPI, restore the original QSPI image from SwissMicros.

<hr /><em>This discussion was created from the release <a href='https://github.com/c3d/DB48X-on-DM42/releases/tag/'>v0.1</a>.</em>

**Comment by balta1980** [comment]
Thank you very much it is amazing what you are doing. I hope you can get some help on this because as I see it, it is a lot of work. 

By my side, due to my lack of knowledge I cannot do too much. But count with my calc and observation capacity. I will communicate all I can think it is of help.

Thank you again for all.
